cookbook/dump-sitemap.rst

```diff
@@ -6,22 +6,57 @@ by so called `SitemapProvider` (see :doc:`sitemap-provider`).
 Each provider returns mostly 50000 links which can return many
 links, which would take a bigger amount of time. The Google bot
 does not wait a long time for the sitemap to be returned.
-Therefore Sulu is able to pre-generate the whole sitemap and
-cache it on the filesystem. This can be triggered by calling the
-following command. This should be be done by a cron-job.
+
+In 99% of the cases the sitemap can be generated on the fly and
+so nothing is required. But for very big websites their is
+a command to pre-generate the sitemap via Sulu and cache it
+on the filesystem.
+This can be triggered by calling the following command. This
+should be done by a cron-job.
+
+.. code-block:: bash
+
+    bin/websiteconsole sulu:website:dump-sitemap
 
 If you use the ``{host}`` replacer in your webspace url
-configuration make sure you have the symfony `router context`_
-parameter configured.
+configuration make sure you have the Symfony ``default_uri``
+configured. Have also a look at the official Symfony
+Documentation about `Generating URLs in Commands`_.
+
+.. code-block:: yaml
+
+    # config/packages/framework.yaml
+    framework:
+        router:
+            default_uri: 'https://example.org'
+
+.. tip::
+
+    You can use ``%env(DEFAULT_URI)%`` to control the configuration
+    via environment variables.
+
+For Symfony Versions before 5.1 you have to configure the
+`router context`_  parameters:
```

Comment:
```suggestion
If you are using a Symfony version before 5.1, you need to configure the
`router context`_  parameters instead of the ``default_uri`` option:
```